package.json

```diff
@@ -21,7 +21,7 @@
 		"@types/jest": "29.5.14",
 		"@types/node": "22.13.11",
 		"aws-cdk": "2.1007.0",
-		"aws-cdk-lib": "2.189.0",
+		"aws-cdk-lib": "2.189.1",
```

Comment:
Charlotte why are we bumping this patch? Aren't we following the CDK dependencies upgrades that the `@guardian/cdk` got for version `61.4.0`?